occameracontrol/agent.py

```diff
@@ -125,3 +125,22 @@ def next_event(self) -> Event:
         if not events:
             return Event('', 0, 0)
         return events[0]
+
+    def verify_agent(self):
+        '''Verify that an agent exists when it is created
+        '''
+        server = config_rt(str, 'opencast', 'server').rstrip('/')
+        username = config_rt(str, 'opencast', 'username')
+        password = config_rt(str, 'opencast', 'password')
+        auth = (username, password)
+        url = f'{server}/capture-admin/agents/{self.agent_id}.json'
```

Comment:
Not sure if this is an error or if `requests` already deals with this automatically. A capture agent identifier may contain characters which need to be URL encoded. For example, you can register an agent called `Test Agent`. That needs to result in a request to `/capture-admin/agents/Test%20Agent.json`.

Do you know if `requests` deals with that case?